src/sentry/notifications/notifications/strategies/role_based_recipient_strategy.py

```diff
@@ -22,8 +22,6 @@ def __init__(self, organization: Organization):
 
     def get_member(self, user: RpcUser) -> OrganizationMember:
         # cache the result
-        if user.class_name() != "User":
-            raise OrganizationMember.DoesNotExist()
```

Comment:
If the type annotation is reliable, this should be a no-op. A glance at the Git history suggests the context has changed.

Reply:
Disregard per `1b0ec600a0e95d1d503ba97696e123c3761ca479`, it turns out we still depend on `class_name` in some non-typesafe places. ☹️